build_tools/buildkite/cmake/android/arm64-v8a/pipeline.yml

```diff
@@ -72,8 +72,6 @@ steps:
       - exit_status: "*"
 
   - label: "test on snapdragon-855 (adreno-640)"
-    # TODO(#4861): Re-enable when phone is fixed
-    skip: "Phone is borked. See https://github.com/google/iree/issues/4861"
```

Comment:
This owns by @stellaraccident and it's disconnected. We can only enable the one in benchmark.yml since that is fixed.

Reply:
The agent config is different, see l.86.

The one in the lab (which is used in benchmark.yml) has cold plate.